posthog/warehouse/models/external_data_schema.py

```diff
@@ -34,6 +34,12 @@ class Status(models.TextChoices):
 
     __repr__ = sane_repr("name")
 
+    @property
+    def is_incremental(self):
+        from posthog.temporal.data_imports.pipelines.schemas import PIPELINE_TYPE_INCREMENTAL_ENDPOINTS_MAPPING
+
+        return self.name in PIPELINE_TYPE_INCREMENTAL_ENDPOINTS_MAPPING[self.source.source_type]
```

Comment:
This is probably fine for now, but I imagine we want to give users the option to have tables/schemas set up as incremental or not in the future